ignite/metrics/accuracy.py

```diff
@@ -126,6 +114,93 @@ def thresholded_output_transform(output):
         device: specifies which device updates are accumulated on. Setting the metric's
             device to be the same as your ``update`` arguments ensures the ``update`` method is non-blocking. By
             default, CPU.
+
+    Examples:
+
+        Binary case
+
+        .. testcode:: 1
+
+            metric = Accuracy()
+            metric.attach(default_evaluator, "accuracy")
+            y_true = torch.Tensor([1, 0, 1, 1, 0, 1])
+            y_pred = torch.Tensor([1, 0, 1, 0, 1, 1])
+            state = default_evaluator.run([[y_pred, y_true]])
+            print(state.metrics["accuracy"])
+
+        .. testoutput:: 1
+
+            0.6666...
+
+        Multiclass case
+
+        .. testcode:: 2
+
+            metric = Accuracy()
+            metric.attach(default_evaluator, "accuracy")
+            y_true = torch.Tensor([2, 0, 2, 1, 0, 1]).long()
```

Comment:
@sdesrozis tensors should be created as `torch.tensor(data)` and not `torch.Tensor(data)` (it was deprecated since v0.4.0). `torch.Tensor` is only used for matching object vs class, e.g. `isinstance(t, torch.Tensor)`.
Please fix that everywhere in doctest strings. Thanks